nomad/structs/acl.go

```diff
@@ -78,6 +79,40 @@ const (
 	// Args: ACLRoleByNameRequest
 	// Reply: ACLRoleByNameResponse
 	ACLGetRoleByNameRPCMethod = "ACL.GetRoleByName"
+
+	// ACLUpsertAuthMethodsRPCMethod is the RPC method for batch creating or
+	// modifying auth methods.
+	//
+	// Args: ACLAuthMethodsUpsertRequest
+	// Reply: GenericResponse
```

Comment:
Thoughts about this, and ACLDeleteAuthMethodsByNameRPCMethod using custom structs rather than the generic?

Reply:
yup, updated comments.